### ISSUE TITLE
 `FirebaseCrashlytics.crash();` throws Error: Method not implemented

### ISSUE THREAD

**Issue by sabahang** [capacitor-community/firebase-crashlytics#17]
**Describe the bug**
Firstly there is a mismatch between Readme and the code. the crash() method doesn't accept any arguments as opposed to an object  in Readme.

Secondly, when I run it without any arguments I get the following error: Method not implemented.
PS: This also happens for FirebaseCrashlytics.recordException method.

I'm running on Android

**Comment by tnoetzel** [comment]
@sabahang - I think that's deliberate. The point of this method is to "force" a crash so it shows up in Crashlytics. If you read the Crashlytics documentation for the native implementations the first step after adding the Crashlytics library to your code is to force a crash; my guess is the `crash()` method is this plugin's way of doing that without forcing developers to mess with any native code.

TLDR: run `FirebaseCrashlytics.crash()` once, then remove it from your app and use the other methods:

```import { Plugins } from "@capacitor/core";
const { FirebaseCrashlytics } = Plugins;

...

FirebaseCrashlytics.setEnabled({
    enabled: true,
});
FirebaseCrashlytics.crash({
    message: "This is a crash message: pretty sure you only need to do this once.",
});
```

**Comment by SalahAdDin** [comment]
@tnoetzel Well, i forced it, i saw it in the console log, but i don't receive the log in the **Firebase Console.**

**Comment by tnoetzel** [comment]
@SalahAdDin - Did you upload your dSYMs as described here? https://github.com/capacitor-community/firebase-crashlytics/issues/2#issuecomment-750934705

**Comment by SalahAdDin** [comment]
@tnoetzel I didn't test it on **iOS**, but it works in **Android**, i tested it.

**Comment by tnoetzel** [comment]
@SalahAdDin - If you're still not seeing crashes in the Firebase Console (sometimes it takes 5 minutes to show up), suggest following the instructions in the ReadMe to enable debugging and see what errors, if any, show up.

**Comment by robingenz** [comment]
I tested it with version 0.3.0 of this plugin on Android. Everything worked. The crash was logged in Firebase. If problems persist, please open a new issue with detailed information to reproduce it.
